frame/assets/src/lib.rs

```diff
@@ -421,7 +425,7 @@ pub mod pallet {
 		///
 		/// Weight: `O(1)`
 		#[pallet::weight(T::WeightInfo::force_create())]
-		pub(super) fn force_create(
+		pub fn force_create(
```

Comment:
@kianenigma Just want to highlight this changes as discussed.

Reply:
Looking at the PR a bit more, can't you use `T::Assets` interface of the AssetFreezer to do this?

Reply:
yeah i dont see why these extrinsics would ever need to be directly callable. The traits should provide access to all the functions you need.